Dockerfile-community

```diff
@@ -0,0 +1,32 @@
+FROM registry.redhat.io/ubi8/ubi:8.6
```

Comment:
alternately this might be from quay ? I do not know the right answer as to pref in community.

Reply:
UBI license is not restrictive and it can be used in open source projects [without requiring a subscription](https://developers.redhat.com/articles/ubi-faq#ubi_details).